docs-ref-conceptual/microsoft-graph-migration.md

```diff
@@ -77,7 +77,7 @@ Command argument and behavior breaking changes are listed in the next section.
 
 ## Known issues
 
-- Regarding generic update arguments, the only supported operation is using `--set` on the root level of a Graph object. When you use `--add`, `--remove` or `--set` on sublevels currently doesn't work (due to the underlying infrastructure change). For unsupported scenarios, you may use `az rest` to directly call [Microsoft Graph API](/graph/api/overview). Examples can be found at https://github.com/Azure/azure-cli/issues/22580.
+- Regarding generic update arguments, the only supported operation is `--set` on the root level of a Graph object. Due to the underlying infrastructure change, the use of `--add`, `--remove` or `--set` on sublevels currently doesn't work. For unsupported scenarios, you may use `az rest` to directly call [Microsoft Graph API](/graph/api/overview). Examples can be found at https://github.com/Azure/azure-cli/issues/22580.
```

Comment:
According to https://e-gmat.com/blogs/due-to-vs-because-of/,

> Use ‘due to’ only to modify nouns.
> Use ‘because of’ to modify verbs.

so we can only use "because of" here.

```suggestion
- Regarding generic update arguments, the only supported operation is `--set` on the root level of a Graph object. Because of the underlying infrastructure change, the use of `--add`, `--remove` or `--set` on sublevels currently doesn't work. For unsupported scenarios, you may use `az rest` to directly call [Microsoft Graph API](/graph/api/overview). Examples can be found at https://github.com/Azure/azure-cli/issues/22580.
```

I am not a native speaker, so may be wrong. 🙂